test/e2e/specs/specs-playwright/wp-likes-spec.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+import {
+	BrowserHelper,
+	DataHelper,
+	LoginFlow,
+	NewPostFlow,
+	GutenbergEditorPage,
+	LikesComponent,
+} from '@automattic/calypso-e2e';
+
+/**
+ * Constants
+ */
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const host = DataHelper.getJetpackHost();
+const viewportName = BrowserHelper.getViewportName();
```

Comment:
Looks like we'll be needing those for each suite. To limit the boilerplate, could we store those values in the suite instance as we currently do with the driver in Selenium, @scinos?

Reply:
See https://github.com/Automattic/wp-calypso/pull/53169#discussion_r639636628 which might reduce the boilerplate here.

Reply:
Selenium suites currently set this boilerplate at the beginning of each file. [Example](https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/specs/specs-calypso/wp-invite-users__editor-spec.js#L29-L34).

Note that I agree it would be nice to remove the boilerplate. I suspect, but am not sure, that at least the `this.timeout` call needs to remain though, because otherwise `mocha` defaults to the 2000ms timeout, for some reason.